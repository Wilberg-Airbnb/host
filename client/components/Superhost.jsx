import React from 'react';
import {List, Item} from 'semantic-ui-react';

const headerStyle = {
  fontWeight: '600'
};

const Superhost = ({name}) => (
  <Item>
    <Item.Content>
    
        <h5>{name.split(' ')[0]} is a Superhost</h5>
      <Item.Description>
      Superhosts are experienced, highly rated hosts who are committed to providing great stays for guests.
      </Item.Description>
    </Item.Content>
  </Item>
)

export default Superhost;