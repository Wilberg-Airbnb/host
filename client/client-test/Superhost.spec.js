import React from 'react';
import Superhost from '../components/Superhost.jsx';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
 
configure({ adapter: new Adapter() });
 
describe('Renders Components', () => {
  let component;
  beforeEach( () => {
    component = shallow(<Superhost name={'Marg Rob'}/>)
  });

  test("should render initial layout", () => {
    const component = shallow(<Superhost name={'Marg Rob'} />);
    expect(component.getElements()).toMatchSnapshot();
  });

});


