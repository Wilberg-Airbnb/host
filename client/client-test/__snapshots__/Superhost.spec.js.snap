// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Components should render initial layout 1`] = `
Array [
  <Item>
    <ItemContent>
      <h5>
        Marg
         is a Superhost
      </h5>
      <ItemDescription>
        Superhosts are experienced, highly rated hosts who are committed to providing great stays for guests.
      </ItemDescription>
    </ItemContent>
  </Item>,
]
`;
